src/api-docs.js

```diff
@@ -1625,6 +1678,95 @@ const apiDoc = {
         },
       },
     },
+    '/wallet/atomic-swap/tx-proposal/fetch': {
+      post: {
+        summary: 'Fetches a proposal data from the Atomic Swap Service',
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            in: 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          }
+        ],
+        requestBody: {
+          description: 'Get the requested proposal by identifier from the atomic swap service.',
+          required: true,
+          content: {
+            'application/json': {
+              schema: {
+                type: 'object',
+                required: ['proposal_id', 'password'],
+                properties: {
+                  proposal_id: {
```

Comment:
You don't have this parameter in the body

Reply:
Fixed it on [595c9a5](https://github.com/HathorNetwork/hathor-wallet-headless/pull/277/commits/595c9a5ffe0cc04e502d7ff1777cd47ba51a4140).